index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <nav class="menu">
+        <h1 class="logo">MIRNA</h1>
+        <img src="./image/Enabled.png" alt="hamburger icon" class="hamburger_icon">
+    </nav>
+    <header class="header">
+        <h2 class="header_title">I'm Mirna, welcome to my portfolio!</h2>
```

Comment:
- [ ] Kindly use `h1` instead of `h2 `because If you want that google (and other search engines) knows about a specific topic, in this case, you, use an` h1` tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.